languagetool-language-modules/en/src/main/java/org/languagetool/rules/en/ContractionSpellingRule.java

```diff
@@ -45,8 +46,8 @@ public Map<String, List<String>> getWrongWords() {
     return wrongWords;
   }
 
-  public ContractionSpellingRule(ResourceBundle messages) {
-    super(messages);
+  public ContractionSpellingRule(ResourceBundle messages, final Language language) {
```

Comment:
You can remove the `final` here. It's correct to have it and in theory it makes the code more clean, but it's also very verbose, so we usually don't use it for parameters.

Reply:
Done!